gui/autofish.lua

```diff
@@ -0,0 +1,110 @@
+-- config ui for autofish
+
+local gui = require("gui")
+local widgets = require("gui.widgets")
+local script = reqscript("autofish")
```

Comment:
I realize that you've named this according to the "plugin" name in `gui/autochop`, but this name conflicts with the `script` module. you're not loading it here, but it could cause confusion for a future reader. I suggest you use "autofish" instead of "script". That's the standard in other scripts that load scripts.